src/System.ComponentModel.TypeConverter/src/project.json

```diff
@@ -1,59 +1,35 @@
 {
-  "frameworks": {
-    "netstandard1.5": {
-      "dependencies": {
-        "System.Collections": "4.0.0",
-        "System.Collections.NonGeneric": "4.0.0",
-        "System.Collections.Specialized": "4.0.0",
-        "System.ComponentModel": "4.0.0",
-        "System.ComponentModel.Primitives": "4.1.0",
-        "System.Diagnostics.Contracts": "4.0.0",
-        "System.Diagnostics.Debug": "4.0.10",
-        "System.Diagnostics.Tools": "4.0.0",
-        "System.Globalization": "4.0.0",
-        "System.Linq": "4.0.0",
-        "System.Reflection": "4.1.0",
-        "System.Reflection.Extensions": "4.0.0",
-        "System.Reflection.TypeExtensions": "4.1.0",
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Runtime.Extensions": "4.0.0",
-        "System.Threading": "4.0.10"
-      },
-      "imports": [
-        "dotnet5.6"
-      ]
-    },
-    "netstandard1.0": {
-      "dependencies": {
-        "System.Collections": "4.0.0",
-        "System.ComponentModel": "4.0.0",
-        "System.ComponentModel.Primitives": "4.1.0",
-        "System.Diagnostics.Contracts": "4.0.0",
-        "System.Diagnostics.Debug": "4.0.0",
-        "System.Diagnostics.Tools": "4.0.0",
-        "System.Globalization": "4.0.0",
-        "System.Reflection": "4.0.0",
-        "System.Reflection.Extensions": "4.0.0",
-        "System.Reflection.Primitives": "4.0.0",
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.0",
-        "System.Runtime.Extensions": "4.0.0",
-        "System.Threading": "4.0.0"
-      },
-      "imports": [
-        "dotnet5.1"
-      ]
-    },
-    "net462": {
-      "dependencies": {
-        "Microsoft.TargetingPack.NETFramework.v4.6.2": "1.0.1"
-      }
-    },
-    "net45": {
-      "dependencies": {
-        "Microsoft.TargetingPack.NETFramework.v4.5": "1.0.1"
-      }
+    "frameworks": {
+        "netstandard1.7": {
+            "dependencies": {
```

Comment:
Same here, I would change this to use the prerelease packages across the board, if you don't do that then you are using older assets instead of the new ones to compile.